frontend/src/actions/TableActions.js

```diff
@@ -500,7 +501,7 @@ function createCollapsedRows({
  * @param {string} tableId
  * @param {array} rows
  */
-function updateCollapsedRows({ tableId, rows }) {
+function updateCollapsedRows({ tableId, rows, changedIds }) {
```

Comment:
document the `changedIds` param, as described above.

also, now that u did into this code,
(loud thinking)
you might consider:
* update the jsdoc summary, if that's not accurate anymore
* and/or change the function name since it feels somehow contradictory with the jsdoc. 
     * jsdoc says creates (and does not mention anything about collapsed things) 
     * method name says update collapsed rows ...... which one is right?